Tests/StoreKitUnitTests/PurchasesOrchestratorTests.swift

```diff
@@ -449,6 +447,51 @@ class PurchasesOrchestratorTests: StoreKitConfigTestCase {
         expect(self.receiptFetcher.receiptDataReceivedRefreshPolicy) == .onlyIfEmpty
     }
 
+    func testPurchaseSK1PackageRetriesReceiptFetchIfEnabled() async throws {
```

Comment:
Functionally equivalent to the removed test below, but uses SK1, since SK2 does not post receipts now.